pkg/cloud/vsphere/services/cloudprovider/csi.go

```diff
@@ -171,7 +171,7 @@ func VSphereCSINodeDaemonSet(storageConfig *cloudprovider.StorageConfig) *appsv1
 					},
 				},
 				Spec: corev1.PodSpec{
-					HostNetwork: true,
+					DNSPolicy: corev1.DNSDefault,
```

Comment:
I'm still curious why we need to use this DNS policy. @divyenpatel

Reply:
No (and this is one of those bad API designs in PodSpec) -- the default value is actually "ClusterFirst" (check query kube-dns). "Default" actually refers to using the node's default nameserver for DNS, but it is not the default value.